turbinia/state_manager_test.py

```diff
@@ -113,3 +114,343 @@ def testStateManagerValidateDataInvalidDict(self, _):
     self.assertNotEqual(test_data['status'], self.test_data['status'])
     self.assertLessEqual(
         len(test_data['status']), state_manager.MAX_DATASTORE_STRLEN)
+
+
+class TestRedisEvidenceStateManager(unittest.TestCase):
+  """Test PSQStateManager class."""
+
+  def _get_state_manager(self):
+    """Gets a Datastore State Manager object for test."""
+    config.STATE_MANAGER = 'Redis'
+    return state_manager.get_state_manager()
```

Comment:
Will need to use the ```fakeredis``` library here to use a fake redis client, otherwise tests will require a local instance of Redis.